packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -184,7 +144,55 @@ export default class SubmodeLayout extends Component<Signature> {
       @onSearch={{this.expandSearchToShowResults}}
       @onCardSelect={{this.handleCardSelectFromSearch}}
     />
+
     <style>
+      .profile-popover {
+        width: 280px;
+        height: 280px;
+        position: absolute;
+        bottom: 68px;
+        left: 20px;
+        z-index: 1;
+        background: white;
+        padding: var(--boxel-sp);
+        flex-direction: column;
+        border-radius: var(--boxel-border-radius);
+        box-shadow: var(--boxel-deep-box-shadow);
+        transition: all 0.1s ease-out;
+        opacity: 0;
+        display: flex;
+      }
+
+      .profile-popover.opened {
+        opacity: 1;
+      }
+
+      .profile-popover-header {
+        font-color: var(--boxel-dark);
```

Comment:
```suggestion
       color: var(--boxel-dark);
```

These styles are duplicated in `profile-info-popover` though, maybe this is a relic of a previous implementation?

Reply:
Nice catch, deleting those